src/mixins/date-mixin.ts

```diff
@@ -211,6 +211,6 @@ export default class DateMixin extends Vue {
    * @returns NaN in case of error
    */
   daysFromToday (date: Date): number {
-    return DateUtilities.daysFromToday(this.getCurrentJsDate, date)
+    return DateUtilities.daysBetweenTwoDates(this.getCurrentJsDate, date)
```

Comment:
Do you have a good reason for doing this?

This code was to map the method in the mixin to the method in the utilities class (in order to keep existing code compatible as we get rid of this mixin).

Reply:
nvm